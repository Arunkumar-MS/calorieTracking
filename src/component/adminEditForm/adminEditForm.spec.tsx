import { jest } from '@jest/globals'
import { shallowToJson } from "enzyme-to-json";
import * as React from 'react';
import { shallow } from 'enzyme';
import AdminFoodEditForm from './index';


describe('AdminFoodEditForm page', () => {
    it('should render properly', function () {
        const wrap = shallow(<AdminFoodEditForm type='add' onClose={() => { }} />)
        expect(shallowToJson(wrap)).toMatchSnapshot();
    })
})
