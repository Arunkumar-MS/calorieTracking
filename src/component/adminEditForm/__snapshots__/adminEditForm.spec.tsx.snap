// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminFoodEditForm page should render properly 1`] = `
<div
  aria-labelledby="modal-title"
  aria-modal="true"
  className="fixed z-10 inset-0 overflow-y-auto text-gray-500"
  role="dialog"
>
  <div
    className="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0"
  >
    <div
      aria-hidden="true"
      className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"
    />
    <span
      aria-hidden="true"
      className="hidden sm:inline-block sm:align-middle sm:h-screen"
    >
      â€‹
    </span>
    <div
      className="inline-block align-bottom bg-white rounded-lg px-4 pt-5 pb-4 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:align-middle w-96 sm:max-w-sm sm:w-full sm:p-6"
    >
      <div
        className="flex -mt-5 -mr-5 justify-end"
        onClick={[Function]}
      >
         
        <CloseIcon
          className="h-10 w-10 fill-current text-gray-400 hover:text-gray-600"
        />
         
      </div>
      <div
        className=""
      >
        <FoodAddForm
          isAdmin={true}
          onSuccess={[Function]}
          type="add"
        />
      </div>
    </div>
  </div>
</div>
`;
